.gitignore

```diff
@@ -1 +1,3 @@
 build_artefacts
+/.DS_Store
```

Comment:
While I think adding the `.DS_Store` stuff is harmless, I don't think a recipe should necessarily be making changes to the code external to the submitted recipe.